src/restoreTerminals.ts

```diff
@@ -107,7 +121,13 @@ async function createNewSplitTerminal(
         break;
       }
       if (numTermsNow > numTermsBefore) {
-        resolve(vscode.window.terminals[numTermsNow - 1]);
+        const newTerm = vscode.window.terminals[numTermsNow - 1];
+
+        if (cwd) {
+          newTerm.sendText(`cd ${cwd}`, true)
```

Comment:
I have two worries here that might be non-issues:
- `cd` _might be_ non-standard on some shells: https://en.wikipedia.org/wiki/Cd_(command)#Implementations
  But should there be a requirement to support the likely nowadays non-existent shell that does not support `cd`, this can be exposed as a config.

- Shell injection via cwd
  I wonder if there is any point of sanitising here, but since this extension already gives equivalent power in specifying the command, this shouldn't be an issue. Related to this, if the path doesn't exist, what is the behavior here?

Reply:
1. I think it's pretty reasonable to expect `cd` to work
2. Yeah, I don't think we need to sanitize
3. This is a valid concern, I've addressed this in my latest commit.